vpr/src/route/rr_graph_area.cpp

```diff
@@ -319,7 +325,9 @@ void count_unidir_routing_transistors(std::vector<t_segment_inf>& /*segment_inf*
      * switches of all rr nodes. Thus we keep track of which muxes we have already
      * counted via the variable below. */
     bool* chan_node_switch_done;
-    chan_node_switch_done = new bool[rr_graph.num_nodes()]();
+    chan_node_switch_done = new bool[rr_graph.num_nodes()];
+    for (auto i = 0; i < rr_graph.num_nodes(); i++)
+        chan_node_switch_done[i] = 0;
```

Comment:
Better to set to false (more clear).
Could recode as vector and use its constructor to set to 0; a bit nicer code as it should get rid of an explicit new and delete too if this is a local variable.